### DISCUSSION TITLE
How to validate a nested input?

### DISCUSSION THREAD

**Discussion by djeymisson-dev** [react-hook-form/react-hook-form#3430] [Q&A] [answered]
I have an input with dot on the name and I can't figure out how to validate it. 

```
<input
	id="name"
	name="user.name"
	type="text"
	aria-invalid={
		errors.name ? 'true' : 'false'
	}
	ref={register({
		required: 'Empty field',
		minLength: {
			value: 2,
			message:
					'Invalid length'
		}
	})}
	/>
	{errors.name && (
		<span className="error">
			{errors.name.message}
		</span>
	)}
```

The errors arrays simply doesn't catch the validation failure and using ```errors.user.name``` returns undefined.

Am I using it wrong?

**Comment by bluebill1049** [comment, marked as answer]
looks correct to me

**Reply by djeymisson-dev** [reply]
Yeah, turned out that it was just the validation triggering that I was calling wrong. Once I fixed that `errors.user?.name` did the trick.

thanks anyway :)
